### PR TITLE
Convert user visible attribute to deleted boolean

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Change column visible in user table from varchar to tinyint
+
+-- 1. Update column visible to store correct tinyint values
+
+UPDATE user SET visible = '0' WHERE visible IS NULL;
```
Comment (Kathrin-Huber) [inline Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql]: what if its not null and not deleted?

Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql]: So far as I know no other values like `null` and `deleted` are possible except somebody changed the values direct in the table.

Comment (Beacze) [inline Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql]: I wanted use this coalesce(trim(visible), '')

Comment (Kathrin-Huber) [inline Kitodo-DataManagement/src/main/resources/db/migration/V2_10__Change_visible_from_varchar_to_tinyint.sql]: We can not exclude, that there MAY BE a change in the database. so we need to check for null, deleted and not deleted.

**Comment by henning-gerhardt** [comment]
There are merge conflicts in four files. Please rebase this branch.

**Pull request merged**